### DISCUSSION TITLE
Scan for vulnerability on container images for the latest releases

### DISCUSSION THREAD

**Discussion by abstractj** [keycloak/keycloak#11066] [Feedback]
The Trivy workflow is configured to perform the scanning of our container images for nightly and legacy builds. Today, we don't have any jobs responsible for performing the scan of `latest` and `17.0.1` (the last release currently) for example.

Should we also include in the scanner the latest releases?

@hmlnarik @stianst @vmuzikar @mposolda @pskopek I'd appreciate your feedback.

**Comment by vmuzikar** [comment]
+1 for adding `latest`. We're already scanning "latest legacy" (`legacy` tag), so I think it'd make sense to have it for Quarkus too.

One argument for NOT adding it would be that we already scan `nightly` tag. However, if there's a CVE in a dependency that we have already updated to a non-affected version in `main`, the scanner won't catch it as our `nightly` is not affected. But our `latest` would be affected as the dependecy is not updated there yet. Therefore IMHO we should scan `latest` as well.
